code/modules/ninja/outfit.dm

```diff
@@ -13,13 +13,13 @@
 	r_pocket = /obj/item/tank/internals/emergency_oxygen
 	internals_slot = SLOT_R_STORE
 	belt = /obj/item/energy_katana
-	implants = list(/obj/item/implant/explosive)
-
+	implants = list(/obj/item/implant/explosive, /obj/item/implant/storage)
 
 /datum/outfit/ninja/post_equip(mob/living/carbon/human/H, visualsOnly = FALSE, client/preference_source)
+	var/obj/item/organ/I = /obj/item/organ/cyberimp/arm/ninja
+	I.Insert(src)
 	if(istype(H.wear_suit, suit))
 		var/obj/item/clothing/suit/space/space_ninja/S = H.wear_suit
 		if(istype(H.belt, belt))
 			S.energyKatana = H.belt
-		S.randomize_param()
-
+		S.randomize_param()
```

Comment:
Don't forget the newline here.

Reply:
Same as before, all files do this for me